src/ParametricOptInterface.jl

```diff
@@ -6,6 +6,8 @@ const MOI = MathOptInterface
 
 const PARAMETER_INDEX_THRESHOLD = 1_000_000_000_000_000_000
 
+@enum ConstraintsInterpretationCode OnlyBounds OnlyConstraints BoundsAndConstraints
```

Comment:
I think JuMP's rule for enums is capitalized
```suggestion
@enum ConstraintsInterpretationCode ONLY_CONSTRAINTS ONLY_BOUNDS  BOUNDS_AND_CONSTRAINTS
```
also only constraints first since it is the default